libs/MeshKernelApi/src/MeshKernel.cpp

```diff
@@ -782,6 +782,32 @@ namespace meshkernelapi
         return lastExitCode;
     }
 
+    MKERNEL_API int mkernel_mesh2d_get_data_reduced(int meshKernelId, Mesh2D& mesh2d)
```

Comment:
should mkernel_mesh2d_get_data be substituted with mkernel_mesh2d_get_data_reduced? if this is possible we have one api less to maintain in the other layers

Reply:
I would only keep mkernel_mesh2d_get_data and mkernel_mesh2d_get_data_reduced. i tested mkernel_mesh2d_get_node_edge_data in grid editor and it does work